package.json

```diff
@@ -16,7 +16,7 @@
     "test": "tap test/*.js"
   },
   "bugs": "http://github.com/creationix/jsonparse/issues",
-  "engines": ["node >= 0.2.0"],
+  "engines": ["node >= 4.8.2"],
```

Comment:
Are we bumping the version here just for one `let` keyword?  Or was there something more I missed also needing newer node?